content/api/metrics.apib

```diff
@@ -106,14 +102,14 @@ The Metrics API uses [JSON Schema](https://json-schema.org/) to define and valid
 Advanced Filter requests that do not adhere to the schema will result in a `400` error response. 
 Specific schema violations will be attached to the `validationErrors` property of the error response.
 The exact schema can be retrieved through the `/deliverability/query-filters-schema` endpoint. 
-See [Advanced Query JSON Schema](#header-advanced-query-json-schema).
+See [Advanced Query JSON Schema](#metrics-get-advanced-query-json-schema).
 
 Please note, advanced filters cannot be used in conjunction with simple filters. Simple filters will be ignored if the `query_filters` parameter is used. 
 e.g. in `?domains=mailing.com&query_filters={}`, the domains query parameter will be ignored. 
 Other query parameters like `from`, `to`, `metrics`, `limit`, etc., will be processed as expected.
 
 #### Groupings Structure 
-The JSON object consists of a top level "groupings" key that has an array of compare objects
+The JSON object consists of a top level "groupings" key that has an array of compare objects.
```

Comment:
I think I meant "root level" when I wrote this 😆 
```suggestion
The JSON object consists of a root level "groupings" array containing compare objects.
```

Reply:
The meaning is probably clear either way but I've made the suggested change.